be/src/runtime/runtime_state.h

```diff
@@ -454,7 +452,10 @@ class RuntimeState {
     // if load mem limit is not set, or is zero, using query mem limit instead.
     int64_t get_load_mem_limit();
 
```

Comment:
add a new method global_runtime_filter_manager to get the query ctx's rf manager.
So that the code is more clear:
runtimestate->local_runtime_filter_manager
runtimestate->global_runtime_filter_manager